functions/lib/emails/booking_approval_email.py

```
@@ -0,0 +1,176 @@
+            "organiserId": organiser_id,
+            "orderId": order_id,
+            "datePurchased": date_purchased,
+            "quantity": quantity,
+            "price": price,
+            "startDate": start_date,
+            "endDate": end_date,
+            "location": location,
+        },
+    }
+
+    response = requests.post(
```

Ruff (0.15.7)

[error] Probable use of `requests` call without timeout

(S113)